packages/app-settings/src/General.tsx

```diff
@@ -75,6 +76,7 @@ class General extends React.PureComponent<Props, State> {
           <div className='medium'>
             <Dropdown
               defaultValue={uiMode}
+              help={t('Adjust the mode from basic (with a limited number of end-user-only apps) to full (with all basic & advanced apps available)')}
```

Comment:
I'd say that both beginners and advances users are end-users. Should we go as far as calling them user friendly? I'd say yes :)
```suggestion
              help={t('Adjust the mode from basic (with a limited number of beginner-user-friendly apps) to full (with all basic & advanced apps available)')}
```